content/blog/blog-2020-08-26-kudo-0.16.0.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2020-08-27
+---
+
+# Announcing KUDO 0.16.0
+
+We are happy to release [KUDO v0.16.0](https://github.com/kudobuilder/kudo/releases/tag/v0.16.0), with focus on upgradability, support for immutable parameters and a lot of other small improvements and bugfixes.
+
+<!-- more -->
+
+To see the full changelog and the list of contributors who contributed to this release, visit [the Github Release](https://github.com/kudobuilder/kudo/releases/tag/v0.16.0) page.
+
+## KUDO Upgrades
+
+Until recently, `kubectl kudo init` would simply apply the expected state to the cluster and ignore any breaking changes that may have been introduced. This is now changed, from now on:
+
+- `kubectl kudo init` will fail if it detects an existing KUDO installation to prevent accidental upgrades.
+- You can use `kubectl kudo init --upgrade` to upgrade an existing installation. You will still have to use the same parameters as with the original installation, this is something we plan to handle in an upcoming release.
```

Comment:
`You will still have to use the same parameters as with the original installation` - I don't understand what it means 🤔 could you be more specific? Parameters meaning kudo image or... ? 🤔

Reply:
Yeah - `--kudo-image`, `--unsafe-self-signed-webhook-ca`, `--service-account`, etc. 

I've removed that sentence here for now, but I've started to work on doc for upgrades in more detail.